File: Basic_Language/1_C/workspace/day02/NumTest.c
//NumTest.c
#include<stdio.h>
void main() {
	//두 정수 입력받고 합 구하기
	int num1 = 0, num2 = 0;
	printf("두 정수를 입력하세요(예 : 10,20) : ");
	scanf_s("%d,%d", &num1,&num2);//10,20 | num1 : 10 / num2 : 20
	printf("두 정수의 합 : %d\n", num1 + num2);
}

File: Basic_Language/2_Python/workspace/day08/NumTest.py
#NumTest.py
while True:
    try:
        num1 = int(input("첫번째 정수 : "))
        num2 = int(input("두번째 정수 : "))
        
        print("결과 :",num1//num2)

    except ValueError:
        print("정수만 입력하세요 제발좀")

    except ZeroDivisionError:
        print("0으로는 나눌 수 없습니다")

    except Exception as e:
        print("알 수 없는 오류 발생 / 개발자에게 알려주세요")
        #print(type(e))
